src/frontend/src/instance/prometheus.rs

```
@@ -96,8 +96,10 @@ impl Instance {
 
             let object_result = if let Some(dist_instance) = &self.dist_instance {
                 let output = futures::future::ready(parse_stmt(&sql))
-                    .and_then(|stmt| {
+                    .and_then(|mut stmts| {
                         let query_ctx = Arc::new(QueryContext::with_current_schema(db.to_string()));
+                        // Only one sql statement supported in this api
+                        let stmt = stmts.remove(0);
```

Should we return an error if there are more than one statements?